### ISSUE TITLE
BigFloats are only returned for objective functions, not variable values

### ISSUE THREAD

**Issue by danspielman** [jump-dev/Convex.jl#426]
If I try the example of optimization using BigFloats:

~~~
using Convex, SDPAFamily
y = Semidefinite(3)
p = maximize(eigmin(y), tr(y) <= sqrt(big(2)); numeric_type = BigFloat)
solve!(p, () -> SDPAFamily.Optimizer(presolve=true))
~~~

The values of the computed variables `y` are not returned as BigFloats

~~~
 typeof(y.value), typeof(p.optval)
(Array{Float64,2}, BigFloat)
~~~

The MOI model does contain the values as BigFloats, and it would be nice to get them:

~~~
using MathOptInterface; const MOI = MathOptInterface
typeof(MOI.get(p.model, MOI.VariablePrimal(), MOI.get(p.model, MOI.ListOfVariableIndices())))
Array{BigFloat,1}
~~~

**Comment by ericphanson** [comment]
Thanks for the report! It should be fixed in Convex v0.14.4 which should be out in an hour or so (i.e. whenever https://github.com/JuliaRegistries/General/pull/31891 is merged).

**Comment by danspielman** [comment]
Thanks!
